app/views/groups/_info_window.html.slim

```diff
@@ -0,0 +1,2 @@
+h2 = group.name
```

Comment:
where does this appear? didn't notice on the shared screenshot

Reply:
This would be a popup that activated on-click of the group marker on the mapbox map